gateway-js/CHANGELOG.md

```diff
@@ -62,6 +62,11 @@ This CHANGELOG pertains only to Apollo Federation packages in the 2.x range. The
 - Adds support for `@interfaceObject` and keys on interfaces [PR #2279](https://github.com/apollographql/federation/pull/2279).
 - Preserves source of union members and enum values in supergraph [PR #2288](https://github.com/apollographql/federation/pull/2288).
 
+## 2.2.3
+
+- Fix possible assertion error during query planning [PR #2299](https://github.com/apollographql/federation/pull/2299)
```

Comment:
Will you delete line 54? It looks like this was released from a separate branch which never made it to `main`, but the PR landed to main and was released as part of 2.3.0 as well. So the same fixes were landed in parallel to 2.2.3 and 2.3.0.